c/include/libsbp/cpp/message_handler.h

```diff
@@ -79,7 +79,7 @@ class CallbackInterface : CallbackInterface<OtherTypes...> {
 
  protected:
   void register_callback(sbp_state_t *state, sbp_msg_callbacks_node_t nodes[]) {
-    sbp_register_unpacked_callback(state,
```

Comment:
Surprised this hadn't already been caught by the compiler, but I guess it at least will be caught now.

Reply:
Curious

Reply:
good spotting, the reason why it might never have been called is because the C++ implementation might have only included 1 type in the template, if you include more than 1, it will fail.